modules/ROOT/pages/runtime-configure-gov.adoc

```diff
@@ -0,0 +1,83 @@
+= To Configure the Mule Runtime to Connect to the Government Cloud
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To use Mule Runtime within the Government Cloud, you must redefine the configuration properties that enable Mule Runtime to point to the EU control plane. You must also provide the Client ID and Client Secret of your environment.
```

Comment:
You may provide either client ID and client secret of an environments belonging to the organization (recommended) or the client ID and secret of the organization itself (or business group).

Reply:
Reworded this a bit - thank you!